addons/common/functions/fnc_getUnitLoadout.sqf

```diff
@@ -1,7 +1,7 @@
 #include "script_component.hpp"
```

Comment:
Probably just get rid of this function and use `CBA_fnc_getLoadout` directly in other places.

Reply:
Went to do that but wasn't sure if you'd want to keep the function around for other uses. I'll remove it 👍